ntoskrnl/mm/section.c

```diff
@@ -4248,6 +4250,12 @@ MmCanFileBeTruncated (IN PSECTION_OBJECT_POINTERS SectionObjectPointer,
         }
 #endif
     }
+    else
+    {
+        DPRINT("ERROR: File can't be truncated because it has references held to its data section\n");
```

Comment:
Is it ok to leave that as DPRINT? Or would it be of interest to promote it to DPRINT1 for some time? (if it's not too noisy during regular ROS usage?)

Reply:
Should be fine with DPRINT1 actually, this case isn't very common.